articles/azure/azs-how-setup-monitoring.md

```diff
@@ -0,0 +1,180 @@
+---
+title: How to configure Azure Monitor for VMs on Azure Stack | UKCloud Ltd
+description: Describes how to configure Azure Monitor for VMs on Azure Stack
+services:
+author: Daniel Brennand
+reviewer:
+lastreviewed:
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Configure Azure Monitor for VMs on Azure Stack
+toc_fullpath: Users/How To/azs-how-setup-monitoring.md
+toc_mdlink: azs-how-setup-monitoring.md
+---
+
+# How to configure Azure Monitor for VMs on Azure Stack | UKCloud Ltd
+
+## Overview
+
+Azure Monitor is the platform service that provides a single source for monitoring Azure resources. With Azure Monitor, you can visualize, query, route, archive, and otherwise take action on the metrics and logs coming from resources in Azure.
+
+This article will enable the following solutions for Azure Stack VMs:
+
+-  [Azure Monitor for VMs](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/vminsights-overview)
+
+## Prerequisites
+
+To complete the steps in this article, you must have appropriate access to a subscription in the Azure** and Azure Stack portal.
+
+## Enabling Azure Monitor for VMs
+
+> [!WARNING]
+> Before proceeding, check [here](https://docs.microsoft.com/en-gb/azure/azure-monitor/insights/vminsights-enable-overview#supported-operating-systems) that your VMs' OS version is supported.
+
+> [!NOTE]
+> The currently supported locations are: *West Central US*, *East US*, *Canada Central*, *UK South*, *West Europe* and *Southeast Asia*.
+
+1. Login to the Azure portal:
+
+    <https://portal.azure.com>
```

Comment:
Can we change this to be in line with how we do it in other docs? E.g. -> https://docs.ukcloud.com/articles/azure/azs-how-identify-global-admins.html